disconnected_guide/index.adoc

```diff
@@ -0,0 +1,579 @@
+# systemctl start httpd
+
+OpenShift Systems
+~~~~~~~~~~~~~~~~~
+
+Build Your Hosts
+^^^^^^^^^^^^^^^^
+
+At this point you can perform the initial build of the hosts that will be part
+of the OpenShift environment. It is recommended to use the latest version of
+Red Hat Enterprise Linux and to perform a minimal installation. You will also
+want to pay attention to the other OpenShift-specific prerequisites, documented
+here:
+
+https://docs.openshift.com/enterprise/3.1/install_config/install/prerequisites.html 
+
+(Overview, System, and Environment Requirements)
+
+Once the hosts are initially built, the repositories can be set up.
+
+Connecting The Repos
+^^^^^^^^^^^^^^^^^^^^
+
+On all of the relevant systems that will need OpenShift software components,
+create the following repository definitions, replacing <ip of the server> with
+the IP or hostname of the Apache server hosting the software repositories (use
+the editor of your choice):
+
+[code]
+# vi /etc/yum.repos.d/ose.repo
+
+Place the following text in the file:
+
+[code]
+[rhel-7-server-rpms]
+name=rhel-7-server-rpms
+baseurl=http://<ip of the server>/repos/rhel-7-server-rpms
+enabled=1
+gpgcheck=0
+[rhel-7-server-extras-rpms]
+name=rhel-7-server-extras-rpms
+baseurl=http://<ip of the server>/repos/rhel-7-server-extras-rpms
+enabled=1
+gpgcheck=0
+[rhel-7-server-ose-3.1-rpms]
+name=rhel-7-server-ose-3.1-rpms
+baseurl=http://<ip of the server>/repos/rhel-7-server-ose-3.1-rpms
+enabled=1
+gpgcheck=0
+
+Host Preparation
+^^^^^^^^^^^^^^^^
+
+At this point, the systems are ready to continue to be prepared following the
+OpenShift documentation. See the following section:
+
+https://docs.openshift.com/enterprise/3.1/install_config/install/prerequisites.html#host-preparation 
+
+Skip the section titled “Registering the Hosts” and start with “Managing Packages”.
+
+OpenShift Installation
+~~~~~~~~~~~~~~~~~~~~~~
+
+Import OpenShift Dockerized Components
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can simply scp the images from the sync server to the individual OpenShift
+hosts:
+
+[code]
+# scp /var/www/html/images/ose3-images.tar root@openshift.host.name:
+# ssh root@openshift.host.name "docker load -i ose3-images.tar"
+
+If you prefer, you could use wget on each OpenShift host to fetch the tar file
+and then perform the Docker import command locally. Perform the same steps for
+the metrics and logging images, if you synchronized them.
+
+On one of the hosts that will act as an OpenShift Master, copy and import the builder images:
+
+[code]
+# scp /var/www/html/images/ose3-builder-images.tar root@openshift.masterhost.name:
+# ssh root@openshift.masterhost.name "docker load -i ose3-builder-images.tar"
+
+Run OpenShift Installer
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You may now follow the rest of the OpenShift installation instructions in the
+documentation. See the following section:
+
+https://docs.openshift.com/enterprise/3.1/install_config/install/advanced_install.html 
+

```

Comment:
@thoraxe I seem to have hit a wall here. Instead of running the advanced installation, I planned on running the quick (which shouldn't make a difference, I think), so the next step here is running `atomic-openshift-installer install` like it says in in the quick installation section. My prospective master host is telling me the command doesn't exist. Is there something missing? I'm guessing something hasn't been installed? Any thoughts?